crates/pop-cli/src/commands/build/mod.rs

```diff
@@ -28,9 +29,9 @@ pub(crate) struct BuildArgs {
 	/// The package to be built.
 	#[arg(short = 'p', long)]
 	pub(crate) package: Option<String>,
-	/// For production, always build in release mode to exclude debug features.
-	#[clap(short, long)]
-	pub(crate) release: bool,
+	/// Build profile [default: debug].
+	#[clap(long, value_enum)]
+	pub(crate) profile: Option<Profile>,
```

Comment:
https://github.com/r0gue-io/pop-cli/blob/fix/build-spec/crates/pop-cli/tests/parachain.rs#L13

Should be updated accordingly to this change.